_authors/osci_2023/zacharyfowler.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Zachary Fowler
+short_name:
```

Comment:
Can you please put your github id into `short_name` too? We just need something in that field.

Reply:
Yes I will do that right now. Thanks for getting back to me.

Reply:
Hello, I just pushed the requested changes. Let me know if you need anything else from me! Thanks!